src/hotspot/os/windows/os_windows.cpp

```diff
@@ -2786,6 +2754,7 @@ LONG WINAPI topLevelExceptionFilter(struct _EXCEPTION_POINTERS* exceptionInfo) {
     }
 
 #if defined(_M_AMD64)
+    extern bool handle_FLT_exception(struct _EXCEPTION_POINTERS* exceptionInfo);
```

Comment:
This seems strange to declare inside another method like this, not a showstopper but it might be better to declare it where handle\_FLT\_exception used to be defined in this file

Reply:
Oh.. I remember wondering where to put this. Meant to come back and find the right header